src/declarative.js

```diff
@@ -134,15 +134,24 @@
       //    By disaling this module write-protection we gain performance.
       //    It could be useful to allow an option to enable or disable this.
       module.locked = true;
-      moduleObj[name] = value;
+
+      // export({name: value})
+      if (typeof name == 'object') {
+        for (var p in name)
```

Comment:
What about `export(new MyObject)`, that has enumerable properties on its prototype. Won't this then copy those over to the module? I'm not to sure it will only receive plain objects, is all.

(If you extend `Object.prototype` I feel you should be on your own)